app/Services/Sync.php

```diff
@@ -307,4 +309,78 @@ public static function getImportStatus() {
     return $totalStatus;
   }
 
+  /**
+   * Updated Post functions
+   */
+
+  /**
+   * Updated Post Scan
+   *
+   * Checks Clock for any posts that have been updated since the function was last run and imports them
+   */
+  public static function updatedPostScan($cli = false) {
+
+    $since = self::getLastUpdatedRunDate();
+
+    if($cli) {
+      WP_CLI::line('Importing posts since last post on '.$since);
+    }
+
+    // Format as unix time
+    $since = strtotime($since);
+
+    // The following gitignored file let's you check when the last scan ran.
+    if(WP_ENV !== 'producton') {
+      $test_run_file = fopen("cron_last_run.tmp", "w") or die("Unable to open file!");
```

Comment:
Can you place this file in to `/tmp/cron_last_run.tmp`? As the project directory isn't writeable on the server. Ta